packages/material-design-icons/README.md

```diff
@@ -0,0 +1,30 @@
+# material-design-icons
+
+This package currently only provides Material icon SVGs required by the Calypso
+nav drawer.  The official `material-design-icons` package (https://github.com/google/material-design-icons)
+is generally quite out-dated.  It also includes many image formats that are not
+relevant to Calypso.
+
+As we adopt Material icons more widely, it will probably be worthwhile to fully
+populate this package with all the available icons.  Alternatively, we could seek
+out an alternative distribution of the icons.
+
+## Using a Material icon in Calypso
```

Comment:
@griffbrad Looks like the readme needs to be updated in a future PR.

Reply:
Thanks for the heads up.  Will post that later today.